src/hazelcore/UHExp.re

```diff
@@ -84,6 +89,10 @@ let case =
     : operand =>
   Case(err, scrut, rules);
 
+let subscript =
+    (~err: ErrStatus.t=NotInHole, body1: t, body2: t, body3: t): operand =>
```

Comment:
instead of body1/2/3, we should give better names to each position: target, start, end